sw/device/lib/testing/entropy_testutils.c

```diff
@@ -61,73 +72,30 @@ void entropy_testutils_auto_mode_init(void) {
           {
               .len = 1, .data = {0x00001003},  // One generate returns 1 block.
           },
-      .reseed_interval = 1,  // Reseed after every generate.
+      .reseed_interval = 4,  // Reseed after every 4 generates.
```

Comment:
Reseeding after every fourth generate (instead of after every generate) is probably fine, especially since this is a test utility function. The comment on Line 65 should be updated too.

Reply:
yeah i did this because the otbn randomness test took FOREVER if i made it 1...
what do you think?